From 5762c8530fe26b602ef1760df3b2e01626ec4cdb Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Wed, 25 Sep 2024 21:59:57 +0200
Subject: [PATCH] Don't build tests when built using pyproject.toml

Just like in the old setup.py let's build dbus-python without tests.
---
 pyproject.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyproject.toml b/pyproject.toml
index 3949d8f..3c4b7cc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -48,3 +48,6 @@ homepage = 'http://www.freedesktop.org/wiki/Software/DBusBindings/#python'
 download = 'http://dbus.freedesktop.org/releases/dbus-python/'
 source = 'https://gitlab.freedesktop.org/dbus/dbus-python/'
 tracker = 'https://gitlab.freedesktop.org/dbus/dbus-python/-/issues'
+
+[tool.meson-python.args]
+setup = ['-Dtests=false']
-- 
2.46.2

